pkg/storage/storagewrappers/doc.go

```diff
@@ -0,0 +1,2 @@
+// Package storagewrappers contains decorators for storage data.
```

Comment:
```suggestion
// Package storagewrappers contains decorators for storage implementations.
```

Reply:
Hmmm it doesn't decorate specific implementations. It decorates an interface